common/djangoapps/student/models_api.py

```diff
@@ -46,3 +47,11 @@ def create_manual_enrollment_audit(
 
 def get_course_enrollment(user, course_run_key):
     return _CourseEnrollment.get_enrollment(user, course_run_key)
+
+
+def get_phone_number(user_id):
+    try:
+        student = UserProfile.objects.get(user_id=user_id)
+        return student.phone_number
```

Comment:
nit: move the return out from the try block to below it.

Reply:
🛠 'd